src/app/Home.jsx

```diff
@@ -0,0 +1,93 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { useState } from 'react';
+import axios from 'axios';
+
+// Allow a user to compare their NI contributions between 2018/19 and 2019/20
+export default function Home() {
+  const [income, setIncome] = useState(0);
+  const [ni2018, setNI2018] = useState(0);
+  const [ni2019, setNI2019] = useState(0);
+
+  const getNationalInsurance = async (runDate) => {
+    const { data } = await axios.post(
+      '/v1/national-insurance',
+      { income },
+      {
+        headers: {
+          'x-run-date': runDate,
+        },
+      }
+    );
+    return data.ni;
+  };
+
+  const compareNationalInsurance = async () => {
+    const niA = await getNationalInsurance('2018-04-06');
+    const niB = await getNationalInsurance('2019-04-06');
+
+    setNI2018(niA);
+    setNI2019(niB);
+  };
+
+  return (
+    <div className="mx-auto max-w-7xl p-8">
+      <div className="mx-auto max-w-3xl">
+        <h1 className="text-4xl">National Insurance Comparer</h1>
+
+        <div className="mt-8">
+          <label htmlFor="income" className="block text-lg font-medium leading-6 text-gray-900">
+            Income
+          </label>
```

Comment:
Good use of `htmlFor` to ensure the label is matched with the input it is labelling which helps with accessibility. 👍 

You may like to indicate that it's the user's _monthly_ income that is expected here (although that's buried in the config of the national insurance bands!)